app/lending/(2-guides)/understanding-liquidations/page.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Soft vs Hard Liquidations
+description: The difference between soft and hard liquidation in Chainflip
+---
+
+## What is a soft liquidation?
+
+Soft liquidation is the first response when your LTV crosses the liquidation threshold.
+
+- Triggered automatically
+- Only a portion of collateral is sold
```

Comment:
I would remove this (the fact that it may be partial is not unique to soft liquidation) and instead add this to the point below:

- Executed in small chunks through DCA-style swaps